tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -972,13 +973,14 @@ cc_library(
         "//tensorflow/compiler/xla:xla_data_proto_cc",
         "//tensorflow/compiler/xla/hlo/ir:hlo",
         "//tensorflow/tsl/platform:statusor",
+        "//tensorflow/tsl/platform:tensor_float_32_hdr_lib",
         "//tensorflow/compiler/xla/stream_executor:stream_executor_headers",
     ] + if_cuda_is_configured([
         "//tensorflow/compiler/xla/stream_executor/cuda:cublas_lt_header",
         "//tensorflow/tsl/platform/default/build_config:cublas_plugin",
         "//tensorflow/compiler/xla/stream_executor:host_or_device_scalar",
         "//tensorflow/compiler/xla/stream_executor:scratch_allocator",
-    ]),
+    ]) + if_static(["//tensorflow/tsl/platform:tensor_float_32_utils"]),
```

Comment:
So I thought this would fix it, because I saw it used elsewhere, but somehow we still get linker errors on tests that depend on this target when compiled without GPU support on Windows. It seems to work if I put the header include inside a #if GOOGLE_CUDA section and move the tensor_float_32_hdr_lib dependency to if_cuda_is_configured block. I will try to reland your change like that.

Reply:
Your proposed solution (here and in the following two comments) seems good to me. It sounds like you already have an internal CL with everything assembled, so I won't update the PR with your changes, but I'm happy to update the PR if that would be helpful.